cmd/template/cluster/provider/capa.go

```diff
@@ -0,0 +1,61 @@
+package provider
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"text/template"
+
+	"github.com/giantswarm/k8sclient/v5/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/kubectl-gs/cmd/template/cluster/provider/templates/aws"
+	"github.com/giantswarm/kubectl-gs/internal/key"
+)
+
+func WriteCAPATemplate(ctx context.Context, client k8sclient.Interface, out io.Writer, config ClusterCRsConfig) error {
+	var err error
+
+	data := struct {
+		Description       string
+		KubernetesVersion string
+		Name              string
+		Namespace         string
+		Owner             string
+		ReleaseVersion    string
+	}{
+		Description:       config.Description,
+		KubernetesVersion: "v1.19.9",
+		Name:              config.Name,
+		Namespace:         key.OrganizationNamespaceFromName(config.Owner),
+		Owner:             config.Owner,
+		ReleaseVersion:    config.ReleaseVersion,
+	}
+
+	for _, t := range aws.GetTemplates() {
+		te := template.Must(template.New(config.FileName).Parse(t))
+		buf := new(bytes.Buffer)
+		// Template from our inputs.
+		err = te.Execute(buf, data)
```

Comment:
```suggestion
		var buf bytes.Buffer
		// Template from our inputs.
		err = te.Execute(&buf, data)
```
According to the docs, "The zero value for Buffer is an empty buffer ready to use". Feels cleaner to me.